crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations/network_dkg.rs

```diff
@@ -399,6 +399,10 @@ fn instantiate_dwallet_mpc_network_encryption_key_public_data_from_dkg_public_ou
                     { NON_FUNDAMENTAL_DISCRIMINANT_LIMBS },
                     secp256k1::GroupElement,
                 >(
+                    Default::default(),
+                    Default::default(),
+                    Default::default(),
+                    Default::default(),
```

Comment:
I think a safer option would be to have `neutral_from_public_parameters().value()` here

Reply:
just do it please